### ISSUE TITLE
Load files without absolute paths

### ISSUE THREAD

**Issue by meneguzzi** [andrellsantos/agentspeak-py#2]
[**main**.py](../__main__.py) currently has the project files not only hardcoded but also referring to absolute paths in your hard drive, please sort out the command line parameters.

Here's how you can do it using [argparse](https://docs.python.org/2/library/argparse.html)

``` python
import argparse

parser = argparse.ArgumentParser(description='Process some integers.')
parser.add_argument('integers', metavar='N', type=int, nargs='+',
                    help='an integer for the accumulator')
parser.add_argument('--sum', dest='accumulate', action='store_const',
                    const=sum, default=max,
                    help='sum the integers (default: find the max)')

args = parser.parse_args()
print args.accumulate(args.integers)
```

So you can run stuff like

``` bash
python prog.py -h
usage: prog.py [-h] [--sum] N [N ...]

Process some integers.

positional arguments:
 N           an integer for the accumulator

optional arguments:
 -h, --help  show this help message and exit
 --sum       sum the integers (default: find the max)
```

and

``` bash
python prog.py 1 2 3 4 --sum
10
```

**Comment by Maumagnaguagno** [comment]
Using hardcoded values is not the problem, one example could be the default MAS if no ``*.maspy`` is provided or found in the current directory.
Since ``>__main__.py file=..\examples\hello-world\helloWorld.maspy`` already works I believe only the hardcoded examples require absolute paths.

I propose this solution:
```Python
import re
import sys
import os # Add this lib
from interpreter import *

if __name__ == '__main__':
    # Use path to autocomplete paths based on this file (__main__.py)
    path = os.path.dirname(os.path.abspath(__file__))
    # file = path + '/../examples/generic/generic.maspy'
    # file = path + '/../examples/hello-world/helloWorld.maspy'
    ...
    # TODO check if MAS file exists here or during ARGV parsing
    interpreter = Interpreter(file, debug, metrics, max_ticks)
    interpreter.run()
```

You need to [watch out for reserved keywords and builtins](http://stackoverflow.com/questions/24942358/is-file-a-keyword-in-python "StackOverfow link about that") being used as variable names, such as **file**.

Argparse is useful as more arguments become available and you do not want to care about ARGV parsing.
But solutions like argparse mix the help text with the argument parsing.
I usually avoid such solutions, you can shift/iterate elements from ARGV as needed while keeping the help readable in the top of the source:

```Python
import sys

HELP = """Agentspeak(py) maspy [debug] [metrics] [max_ticks integer]
    maspy     - maspy file
    debug     - activate debug, default: disabled
    metrics   - activate metrics, default: disabled
    max_ticks - execute integer ticks, default: unlimited"""

maspy = 'default/file.maspy'
debug = False
metrics = False
max_ticks = None

argv = sys.argv
argv.pop(0)
# Pop position arguments here if any
while argv:
    arg = argv.pop(0)
    if arg == 'file':
        try:
            maspy = argv.pop(0)
        except:
            print('maspy expected an argument')
            sys.exit(1)
    elif arg == 'debug':
        debug = True
    elif arg == 'metrics':
        metrics = True
    elif arg == 'max_ticks':
        try:
            max_ticks = int(argv.pop(0))
        except:
            print('max_ticks expected an integer argument')
            sys.exit(1)
    else:
        print('Unknown argument: ' + arg)
        print(HELP)
        sys.exit(1)
```

Note that debug and metrics need no value, the feature is set to true if they are present, otherwise you need one more shift.
Position arguments can be obtained between the first ``argv.pop(0)`` and the loop.

**Comment by Maumagnaguagno** [comment]
I believe this issue can be closed since commit 3cbb49964349ef6a427b73ce17854e8da2fba431.
